lib/os.js

```diff
@@ -144,19 +143,60 @@ function endianness() {
 }
 endianness[Symbol.toPrimitive] = () => kEndianness;
 
+function countOnes(n) {
+  let count = 0;
+  while (n !== 0) {
+    n = n & (n - 1);
+    count++;
+  }
+  return count;
+}
+
+function getCIDR({ address, netmask, family }) {
+  let ones = 0;
+  let split = '.';
+  let range = 10;
+  let groupLength = 8;
+  let hasZeros = false;
+
+  if (family === 'IPv6') {
+    split = ':';
+    range = 16;
+    groupLength = 16;
+  }
+
+  const parts = netmask.split(split);
+  for (const part of parts) {
+    if (part !== '') {
+      const binary = parseInt(part, range);
+      const tmp = countOnes(binary);
+      ones += tmp;
+      if (hasZeros) {
+        if (tmp !== 0) {
+          return null;
+        }
+      } else if (tmp !== groupLength) {
+        if (binary % 2 !== 0) {
```

Comment:
You could use `(binary & 1) !== 0`. Last time I tried the reminder operator was not optimized for this, maybe it is now, don't know.

Reply:
Tracking bug https://bugs.chromium.org/p/v8/issues/detail?id=8069

Reply:
Fixed in 6.0: https://crrev.com/35974e2da664fdbc5678b540e2ab0c64b1bbd182